src/base/dof_map_constraints.C

```diff
@@ -1717,7 +1717,7 @@ dof_id_type DofMap::n_local_constrained_dofs() const
   const DofConstraints::const_iterator lower =
     _dof_constraints.lower_bound(this->first_dof()),
     upper =
-    _dof_constraints.upper_bound(this->end_dof()-1);
+    _dof_constraints.lower_bound(this->end_dof());
```

Comment:
This looks like an actual bugfix... was that suggested by the sanitizer?

Reply:
It wasn't suggested by the sanitizer; that only reports filename and line number and what type of behavior was detected, and leaves it up to the developer to figure out how to fix it.  Maybe in LLVM 30.

But ... yeah, this looks like a real bug fix.  I recently spent a ton of time fixing a bug that was triggered by an unexpected corner case with element deletion, just like this one could easily have been, and as annoying as the sanitizer is I have to say this fix alone made me feel like the annoyance was worth it so far.